doc/RelNotes.md

```diff
@@ -0,0 +1,23 @@
+Release Notes {#mainpage}
+============
+
+[TOC]
+
+# Introduction {#Introduction}
+
+Welcome to RESTBridge, a library that provides an simple way to expose a REST
```

Comment:
Yes, but the README is what is shown on the project page in Github. And only the introduction is repeated. Seems fine to me :-/